src/components/VarianceTypeActionsPopover.vue

```diff
@@ -4,14 +4,8 @@
       <ion-list-header>
         {{ translate("Variance type") }}
       </ion-list-header>
-      <ion-item button>
-        {{ translate("No variance") }}
-      </ion-item>
-      <ion-item button>
-        {{ translate("Single variance") }}
-      </ion-item>
-      <ion-item button lines="none">
-        {{ translate("All variance") }}
+      <ion-item lines="none" button @click="updateVarianceType(type)" v-for="type in rejectReasonEnumTypes" :key="type.enumTypeId">
+        {{ type.enumTypeId }}
```

Comment:
Display description in place of id

Reply:
Based on dicussion with @dt2patel sir, we are showing id for now as description is too long doesn't look good in the UI.